src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -176,6 +176,10 @@ export class HTMLMarkdownConverter {
 			replacement: function (content, node, options) {
 				// For elements that aren't lists, convert <br> into its markdown equivalent
 				if (node.parentElement?.nodeName !== 'LI') {
+					// Keeps <br> in table cell to create new line in the table cell
+					if (node.parentElement?.nodeName === 'TD') {
```

Comment:
Does this work with all table headers?

Reply:
will have to add another condition for table headers as well, this condition was for table cells.